data/announcements/scheduled.yaml

```
@@ -226,7 +226,9 @@ announcements:
     color: #fffff;
   title: |
     <img src ="/images/announcements/kccnc-eu-2024-white.svg" style="float: right; height: 80px;" />
-    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/register/?utm_source=kubernetes&utm_medium=all&utm_campaign=kubeconeu24&utm_content=slim-banner">KubeCon + CloudNativeCon EU 2024</a> <em> Paris, France + Virtual</em>.
+    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">Kubecon + CloudNativeCon Europe 2024</a>
```

Link is still pointing to the North America event. Can you update to the EU landing page?

```suggestion
    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">Kubecon + CloudNativeCon Europe 2024</a>
```